tool/gocross/autoflags.go

```diff
@@ -47,6 +47,11 @@ func autoflagsForTest(argv []string, env *Environment, goroot, nativeGOOS, nativ
 		subcommand = argv[1]
 	}
 
+	// avoid trimpath in test, to avoid hitting IDE/LSP/caching bugs.
```

Comment:
which bug(s)? I don't like vague comments like this where you can't follow up to see if the bug's been fixed and whether this workaround is still necessary.

Reply:
I just removed the comment, this isn't meant to imply one specific bug, but the class of bugs that regularly recur in integrations.

Reply:
It still looks super mysterious and not obvious without a comment.

Presumably trimpath was there for a reason (hiding path details in release builds? cache hit sharing?) and there's a reason we don't want it in tests (so you can click an error message stack trace and jump to the right file in more editors?).

But does changing trimpath behavior between modes reduce the cache sharing we get between the two halves of the world? e.g. if one builder does `go install ./cmd/...` and another build later does `go test ./foo`, those `foo` tests can't use package builds that were built with `-trimpath`, as the cache signature will be different. Do we not care about that extra work?